src/main/java/org/opensearch/security/dlic/rest/api/RolesApiAction.java

```diff
@@ -42,9 +41,13 @@
 import java.util.List;
 import java.util.Map;
 
+import static org.opensearch.security.dlic.rest.api.RequestHandler.methodNotImplementedHandler;
 import static org.opensearch.security.dlic.rest.support.Utils.addRoutesPrefix;
 
-public class RolesApiAction extends PatchableResourceApiAction {
+public class RolesApiAction extends AbstractApiAction {
+
+    protected final static String RESOURCE_NAME = "role";
```

Comment:
Is it possible to use the CType enum here? 

```
protected final static String RESOURCE_NAME = CType.ROLES.toLCString();
```

Reply:
nevermind, I see that this is singular

Reply:
Ahhh yes I tried to use `CType.ROLES.toLCString()` as a resource name but the response in this case looke lime this: `roles 'aaaaa' not found`. I will remve it and move resource name to the validator.